content/en/docs/use-cases/openshift.md

```diff
@@ -44,18 +44,44 @@ oc login -u kubeadmin -p <your password> https://api.crc.testing:6443
 
 ### Security Context Constraints
 
-Before installing Flux, you need to set the **anyuid** SCC for Kustomization and Helm controllers,
-and the **privileged** SCC for the other 4 controllers in the `flux-system` namespace, like this:
+Before installing Flux with CLI, you need to set the **nonroot** SCC for all controllers in the `flux-system` namespace, like this:
 
 ```sh
 NS="flux-system"
-oc adm policy add-scc-to-user anyuid system:serviceaccount:$NS:kustomize-controller
-oc adm policy add-scc-to-user anyuid system:serviceaccount:$NS:helm-controller
+oc adm policy add-scc-to-user nonroot system:serviceaccount:$NS:kustomize-controller
```

Comment:
Does this work if the namespace and service accounts are not present on the cluster?

Reply:
yes